.travis.yml

```diff
@@ -20,6 +20,20 @@ notifications:
 cache:
     apt: true
     pip: true
+    directories:
```

Comment:
What is this doing?

Reply:
It is an attempt to cache the packages that Conda installs. I just had a couple of Travis runs where downloading MKL from within Conda (~200MB took, ~10 Minutes) and stretchted the time of the whole test run to ~16. It would be great if we could cut down on dependencies/or as a workaround make at least the Travis setup faster. This route does not help, but we should find something else.